### PR TITLE
add file RnaTranscription.java

### PR DESCRIPTION
**Pull request by nngrz**
[Java exercise: RnaTranscription](https://exercism.org/tracks/java/exercises/rna-transcription/edit)

I used the solution with a helper method to translate DNA to RNA and use Switch expression

### CODE REVIEW

File: src/exercism/RnaTranscription.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class RnaTranscription {
+
+    String transcribe(String dnaStrand) {
+        String result = "";
+        for (int i = 0; i < dnaStrand.length(); i++) {
+            char c = dnaStrand.charAt(i);
+            if (c == 'G') {
```
Comment (balinskia) [inline src/exercism/RnaTranscription.java]: Please move the logic from line 7-17 to a new static function 'transformChar'

**Review by balinskia — CHANGES_REQUESTED** [review]
Code looks good. One cosmetic change

File: src/exercism/RnaTranscription.java (hunk 1 above)
Comment (nngrz) [reply src/exercism/RnaTranscription.java]: please look at my PR, that's what am I doing

File: src/exercism/RnaTranscription.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            case 'T' -> 'T';
+            case 'A' -> 'U';
+            default -> throw new IllegalArgumentException("Invalid DNA nucleotide: " + dnaNucleotide);
+        };
+    }
+
+    String transcribe(String dnaStrand) {
+        String result = "";
+        for (char c : dnaStrand.toCharArray()) {
+            result = result + DNAToRNATranslator(c);
```
Comment (balinskia) [inline src/exercism/RnaTranscription.java]: Please find out why using a string builder might be better here

**Review by balinskia — APPROVED** [review]

File: src/exercism/RnaTranscription.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    public static void main(String[] args) {
+        RnaTranscription rnaTranscription = new RnaTranscription();
+        // test1: empty string
+        System.out.println(rnaTranscription.transcribe(""));
+        // test2: single character
+        System.out.println(rnaTranscription.transcribe("G"));
+        // test3: over 1 characters
+        System.out.println(rnaTranscription.transcribe("GCTA"));
+        // test4: given string has chracter not match "G""C""T""A"
```
Comment (balinskia) [inline src/exercism/RnaTranscription.java:29]: Very good tests!

**Review by balinskia — APPROVED** [review]

**Pull request merged**